.github/workflows/ci.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10', '3.11']
-        django: ['3.2', '4.1', '4.2']
+        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        django: ['3.2', '4.2']
         exclude:
-          - python: '3.11'
+          - python: '3.12'
             django: '3.2'
```

Comment:
django 3.2 doesn't support python 3.11, so both 3.11 and 3.12 need to be excluded